package.json

```diff
@@ -29,7 +29,17 @@
   },
   "homepage": "https://github.com/andela/valinor-ah-frontend#readme",
   "devDependencies": {
+    "@babel/core": "^7.1.6",
```

Comment:
Are all of these changes required?

Reply:
@augustineezinwa This doesn't seem to have been done or answered.

Reply:
Let me fix that.

Reply:
@mbilesanmi  done